src/components/ArchivedReportFooter.js

```diff
@@ -44,40 +32,14 @@ const propTypes = {
     ...withLocalizePropTypes,
 };
 
-const defaultProps = {
-    reportClosedAction: {
-        originalMessage: {
-            reason: CONST.REPORT.ARCHIVE_REASON.DEFAULT,
-        },
-    },
-};
-
 const ArchivedReportFooter = (props) => {
-    const archiveReason = lodashGet(props.reportClosedAction, 'originalMessage.reason', CONST.REPORT.ARCHIVE_REASON.DEFAULT);
-    let displayName = lodashGet(props.personalDetails, `${props.report.ownerEmail}.displayName`, props.report.ownerEmail);
-
-    let oldDisplayName;
-    if (archiveReason === CONST.REPORT.ARCHIVE_REASON.ACCOUNT_MERGED) {
-        const newLogin = props.reportClosedAction.originalMessage.newLogin;
-        const oldLogin = props.reportClosedAction.originalMessage.oldLogin;
-        displayName = lodashGet(props.personalDetails, `${newLogin}.displayName`, newLogin);
-        oldDisplayName = lodashGet(props.personalDetails, `${oldLogin}.displayName`, oldLogin);
-    }
-
+    const archivedText = ReportUtils.getArchivedText(props.report, props.reportActions, props.personalDetails, props.policies) || CONST.REPORT.ARCHIVE_REASON.DEFAULT;
```

Comment:
I think it would be cleaner to have the default reason set in `ReportUtils.getArchivedText`, especially since it is already used there.